### ISSUE TITLE
关于多查询模式

### ISSUE THREAD

**Issue by LiuYT0929** [mode-str/crossview#8]
您好，请问对于University-1652数据集在推理时如何使用多查询模式呢？

if opt.multi:
    image_datasets = {x: datasets.ImageFolder( os.path.join(data_dir,x) ,data_transforms) for x in ['gallery','query','multi-query']}
    dataloaders = {x: torch.utils.data.DataLoader(image_datasets[x], batch_size=opt.batchsize, shuffle=False) for x in ['gallery','query','multi-query']}

'gallery','query','multi-query'这几个数据子集在University-1652数据集中应该是对应什么内容呢？
非常期待并感谢您的答复！

**Comment by mode-str** [comment]
本文并未针对多查询模式开展实验，该数据集也并未提供用于多查询模式的数据划分，但多查询模式相较于单一图片查询通常被认为具有更好的性能。如果要使用多查询模式，你可以自行针对该数据集进行数据划分，如果你要使用本文模型，那可能还需要调整网络架构或针对网络的输出结果进行一些操作，以便多查询图像的特征可以进行融合或其余操作。
